src/java/org/apache/cassandra/db/streaming/CassandraStreamReceiver.java

```diff
@@ -190,11 +192,11 @@ private boolean cdcRequiresWriteCommitLog(ColumnFamilyStore cfs)
      * For CDC-enabled tables and write path for CDC is enabled, we want to ensure that the mutations are
      * run through the CommitLog, so they can be archived by the CDC process on discard.
      */
-    private boolean requiresWritePath(ColumnFamilyStore cfs)
+    public boolean requiresWritePath(ColumnFamilyStore cfs)
```

Comment:
Looks like this is only used externally by `CassandraStreamReceiverTest`, so suggest changing to default visibility and use `@VisibleForTesting` (which would need to be imported as well).
```suggestion
    @VisibleForTesting
    boolean requiresWritePath(ColumnFamilyStore cfs)
```